src/js/components/charts/TimeSeriesChart.js

```diff
@@ -79,7 +80,7 @@ var TimeSeriesChart = React.createClass({
     // happens after mount and ends up keeping the axis code outside of react
     // unfortunately.
     // If non `data` props change then we need to update the whole graph
-    if (!_.isEqual(_.omit(props, 'data'), _.omit(nextProps, 'data'))) {
+    if (!_.isEqual(Util.omit(props, 'data'), Util.omit(nextProps, 'data'))) {
```

Comment:
_.isEqual is the most horrifying thing ever, I almost feel like we should keep _ in the repo just for _.isEqual